users/tests.py

```diff
@@ -1381,7 +1422,20 @@ def setUp(self) -> None:
             'exp': 1664703860
         }
         self.assessor_company = Company.objects.create_user(email='company@company.com', password='Password123')
+        self.client = APIClient()
```

Comment:
Hello there, Indi. I'm not sure what's being tested in this test. Should any assertions or comparisons be made?